src/main/java/com/revature/domain/EvalType.java

```diff
@@ -35,39 +38,53 @@ public String getDescription() {
 	public void setDescription(String description) {
 		this.description = description;
 	}
+	
+
+	public boolean isActive() {
+		return isActive;
+	}
+
+	public void setisActive(boolean isActive) {
+		this.isActive = isActive;
+	}
 
 	@Override
 	public int hashCode() {
-		int hash = 3;
-		hash = 19 * hash + Objects.hashCode(this.id);
-		hash = 19 * hash + Objects.hashCode(this.description);
-		return hash;
+		final int prime = 31;
+		int result = 1;
+		result = prime * result + ((description == null) ? 0 : description.hashCode());
+		result = prime * result + ((id == null) ? 0 : id.hashCode());
+		result = prime * result + (isActive ? 1231 : 1237);
+		return result;
 	}
 
 	@Override
 	public boolean equals(Object obj) {
-		if (this == obj) {
+		if (this == obj)
 			return true;
-		}
-		if (obj == null) {
+		if (obj == null)
+			return false;
+		if (getClass() != obj.getClass())
 			return false;
-		}
-		if (getClass() != obj.getClass()) {
+		EvalType other = (EvalType) obj;
+		if (description == null) {
+			if (other.description != null)
+				return false;
+		} else if (!description.equals(other.description))
 			return false;
-		}
-		final EvalType other = (EvalType) obj;
-		if (!Objects.equals(this.description, other.description)) {
+		if (id == null) {
+			if (other.id != null)
+				return false;
+		} else if (!id.equals(other.id))
 			return false;
-		}
-		if (!Objects.equals(this.id, other.id)) {
+		if (isActive != other.isActive)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1126)